ember-core/shared/src/main/scala/org/http4s/ember/core/h2/H2Server.scala

```diff
@@ -305,8 +305,10 @@ private[ember] object H2Server {
     def processCreatedStreams(h2: H2Connection[F]): F[Unit] =
       Stream
         .fromQueueUnterminated(h2.createdStreams)
-        .map(i => Stream.eval(processCreatedStream(h2, i).attempt))
-        .parJoin(localSettings.maxConcurrentStreams.maxConcurrency)
+        .parEvalMap(localSettings.maxConcurrentStreams.maxConcurrency)(i =>
```

Comment:
Since we are discarding the results, this can/should be `parEvalMapUnordered` I think.